conda_forge_tick/git_utils.py

```diff
@@ -405,10 +401,7 @@ def _munge_dict(dest, src, keys):
 
 
 def lazy_update_pr_json(
-    pr_json: Union[Dict, LazyJson],
-    ctx: GithubContext,
-    force: bool = False,
-    trim: bool = True,
```

Comment:
Please back out the changes to the keyword here.

Reply:
Why? `trim` is never set to anything other than the default value `True`. If trim is set to `False`, the code below crashes.

Reply:
This pr mixes typing changes with functional ones. I can't tell if things will break. I'm all for the cleanup but I need it in smaller chunks so that it can be reviewed properly.

Reply:
Moved to #2273.